synth.py

```diff
@@ -22,27 +22,25 @@
 logging.basicConfig(level=logging.DEBUG)
 
 # Run the gapic generator
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 versions = ['v1', 'v1p1beta1', 'v1p2beta1', 'v1p3beta1', 'v1p4beta1']
 for version in versions:
-    library = gapic.node_library('vision', version)
-    s.copy(library, excludes=['src/index.js', 'README.md', 'package.json'])
+    library = gapic.typescript_library(
+        'vision', version,
+        generator_args={
+            "grpc-service-config": f"google/cloud/vision/{version}/vision_grpc_service_config.json",
+            "package-name": f"@google-cloud/vision",
+            },
+        proto_path=f'/google/cloud/vision/{version}',
+        extra_proto_files=['google/cloud/common_resources.proto'],
+    )
+    s.copy(library, excludes=['src/index.ts', 'README.md', 'package.json', 'tslint.json'])
```

Comment:
Why exclude the `README`, or `tslint.json`?

Reply:
We do need to exclude `README` (the one generated by gapic-generator-typescript does not make much sense!) but not `tslint.json` I guess.

We'll work on making this list of files hardcoded in `synthtool` so that we had less code in `synth.py` across libraries.

Reply:
(or maybe we don't really need to exclude `README` since it will be overwritten later - is that what you meant?)

Reply:
Exactly.  We shouldn't have a custom README unless we really have to.

Reply:
got it, unskip two files now.